tests/testthat/test-dplyr.R

```diff
@@ -27,7 +27,8 @@ test_that("dplyr::mutate works as expected", {
   expect_print_matches_file(input, "dplyr/mutate-skim.txt")
 
   no_variable <- dplyr::mutate(skimmed_iris, skim_variable = NULL)
-  expect_print_matches_file(no_variable, "dplyr/mutate-no-skim.txt")
+  identical(capture.output(print(no_variable)), 
+            capture.output(print(tibble::as_tibble(no_variable))))
```

Comment:
I have a small style comment. Then feel free to merge.

```suggestion
identical(
  capture.output(print(no_variable)), 
  capture.output(print(tibble::as_tibble(no_variable)))
)
```